app/app.go

```diff
@@ -35,6 +35,7 @@ import (
 const (
 	appName          = "IrisApp"
 	FlagReplayHeight = "replay_height"
+	FlagReplay       = "replay"
```

Comment:
So confused with the flag `replay_height `

Reply:
What's the difference between replay and replay_height?  @chengwenxi

Reply:
`replay_height` mean iris replay to  appointed height and then software stop for iris export.
`replay` mean iris and tendermint replay last block and then iris will start. @haifengxi

Reply:
All iricli flags have dash "-" instead of underscore "_" in their names, shouldn't we follow the same convention for other commands such as iris? @chengwenxi

Reply:
Let's review the code tomorrow.

Reply:
Yes, we should we follow the same convention for all commands.